javascript/ql/test/query-tests/Security/CWE-611/libxml.noent.js

```diff
@@ -5,3 +5,13 @@ express().get('/some/path', function(req) {
   // NOT OK: unguarded entity expansion
   libxmljs.parseXml(req.param("some-xml"), { noent: true });
 });
+
+express().post('/some/path', function(req, res) {
```

Comment:
Could we add a `noent:false` test case?
Is `noblanks:xxx` irrelevant? If so, can we drop it?

Reply:
> Is noblanks:xxx irrelevant? If so, can we drop it?

It is, and we can.